assets/i18n/en.toml

```diff
@@ -106,21 +106,21 @@ remaining = "Remaining"
 [main.loadpointSettings]
 title = 'Settings "{0}"'
 vehicle = "Vehicle"
-currents = "Charging"
+currents = "Charging current"
 default = "default"
 disclaimerHint = "Note:"
-disclaimerText = "Changes are not persistent yet. They will be reset after server restart."
+disclaimerText = "Changes will be lost after the server is restarted."
 
 [main.loadpointSettings.minSoC]
-label = "Minimal SoC"
-description = 'Range for emergencies. Vehicle gets "fast" charged to {0}% in PV mode. Then continues with PV surplus only.'
+label = "Minimal state of charge"
+description = 'For emergencies. The vehicle gets "fast" charged to {0}% from solar, and then continues with only solar surplus.'
```

Comment:
```suggestion
description = 'For emergencies. The vehicle gets "fast" charged to {0}% from solar, and then continues with only the solar surplus.'
```

Reply:
Imho the description is wrong. Its not first this then that. It is "immedialy full blast", independent of the power source.

Reply:
again "," => "solar and then continues" ??